.tool-versions

```diff
@@ -0,0 +1,6 @@
+# This file pins the required versions of runtime dependencies for the oracle-toolkit.
+# It is used by the build script to generate an accurate SBOM and ensures a
+# consistent environment for developers.
+
+ansible 2.16.5
```

Comment:
The current minimum Ansible version is in https://github.com/google/oracle-toolkit/blob/5d6dfa766c090f1c869a23b4432a270cfcedacf3/group_vars/all.yml#L167

This can change from time to time though;  in the spirit of automatic SBOM scanning, is there any way to fetch it from that canonical location?